ensembl/conf/ini-files/oreochromis_niloticus.ini

```diff
@@ -67,6 +67,6 @@ dsn = 0
 
 
 VEP_HGVS          = ENSONIT00000003039.1:c.4G>C\nENSONIT00000011538.1:c.43+16C>T\nENSONIT00000000406.1:c.5delC
-VEP_VCF           = GL831133.1 11647 . G C . . .\nGL831134.1 313076 . C T . . .\nGL831135.1 33505 . GC G . . .
-VEP_ENSEMBL       = GL831133.1 11647 11647 G/C 1\nGL831134.1 313076 313076 C/T 1\nGL831135.1 33506 33506 C/- 1
+VEP_VCF           = LG1 27035859 . G C . . .\nLG13 15773411 . GC C . . .\nLG18 33760886 . C T . . .
+VEP_ENSEMBL       = LG1 27035859 27035859 G/C 1\nLG13 15773412 15773412 C/- 1\nLG18 33760886 33760886 C/T 1
 VEP_PILEUP        = GL831133.1 11647 G C\nGL831134.1 313076 C T\nGL831135.1 33505 * -C
```

Comment:
VEP_PILEUP is outdated and can be removed.

Reply:
Could you please an example variant like we have for vevet: https://github.com/Ensembl/public-plugins/blob/master/ensembl/conf/ini-files/chlorocebus_sabaeus.ini#L75-L76
Maybe:
VARIATION_PARAM = LG19:19971155:G_A:PRJEB38548;vf=LG19:19971155:G_A:PRJEB38548
VARIATION_TEXT = LG19:19971155:G_A:PRJEB38548

Reply:
I removed VEP_PILEUP and added the variation examples.